### DISCUSSION TITLE
Pixel watch LTE as standalone (AKA no-phone) uploader?

### DISCUSSION THREAD

**Discussion by kretch** [NightscoutFoundation/xDrip#2379] [Q&A]
Howdy all
Long time user, first-time-in-awhile question:
- Google released the new Pixel phone with LTE- do you think this could be used as a standalone uploader (from a Dexcom G6) into nightscout?

TIA
Yaron

**Comment by tzachi-dar** [comment]
Theoretically, it might be.
However, this needs some work from the perspective of:
1) Add code to upload data to nightscout from the watch (AFAIK we don't have such code right now).
2) We need to understand how the watch behaves when it comes to battery saving. For example, I have noticed that the galaxy watch 4, doesn't obey the specs when it comes to tymers. We need to understand if the watch can actually collect data from the dexcom.
